test/integration/controlplane/apiserver_identity_test.go

```diff
@@ -99,13 +99,15 @@ func TestCreateLeaseOnStart(t *testing.T) {
 }
 
 func TestLeaseGarbageCollection(t *testing.T) {
+	// This shortens the GC check period to make the test run faster.
+	// Since we are testing GC behavior on leases we create, what happens to
+	// the real apiserver lease doesn't matter.
+	controlplane.IdentityLeaseDurationSeconds = 1
+	controlplane.IdentityLeaseGCPeriod = time.Second
+	controlplane.IdentityLeaseRenewIntervalPeriod = time.Second
```

Comment:
Can you set them back when you're done to avoid weird things in other tests?

Reply:
Good catch -- updated